src/pudl/etl/epacems_assets.py

```diff
@@ -107,30 +115,36 @@ def consolidate_partitions(context, partitions: list[YearPartitions]) -> None:
         context: dagster keyword that provides access to resources and config.
         partitions: Year and state combinations in the output database.
     """
-    partitioned_path = PudlPaths().output_dir / "core_epacems__hourly_emissions"
-    monolithic_path = PudlPaths().output_dir / "core_epacems__hourly_emissions.parquet"
+    partitioned_path = _partitioned_path()
+    # Create monolithic output in base output directory and parquet directory
+    monolithic_paths = [PudlPaths().output_dir, PudlPaths().output_dir / "parquet"]
     schema = Resource.from_id("core_epacems__hourly_emissions").to_pyarrow()
 
-    with pq.ParquetWriter(
-        where=monolithic_path, schema=schema, compression="snappy", version="2.6"
-    ) as monolithic_writer:
-        for year_partition in partitions:
-            for state in EPACEMS_STATES:
-                monolithic_writer.write_table(
-                    # Concat a slice of each state's data from all quarters in a year
-                    # and write to parquet to create year-state row groups
-                    pa.concat_tables(
-                        [
-                            pq.read_table(
-                                source=partitioned_path
-                                / f"epacems-{year_quarter}.parquet",
-                                filters=[[("state", "=", state.upper())]],
-                                schema=schema,
-                            )
-                            for year_quarter in year_partition.year_quarters
-                        ]
+    for year_partition in partitions:
+        for state in EPACEMS_STATES:
+            for base_path in monolithic_paths:
+                monolithic_path = base_path / "core_epacems__hourly_emissions.parquet"
+                with pq.ParquetWriter(
+                    where=monolithic_path,
+                    schema=schema,
+                    compression="snappy",
+                    version="2.6",
+                ) as monolithic_writer:
```

Comment:
I ran the full ETL locally overnight and the quarterly outputs look as expected, but both copies of the consolidated outputs only contain a single state-year of data: Alaska, 2023 (through Q3 since that's as much data is available in 2023 currently). So I wonder if this isn't overwriting the file entirely rather than appending new row-groups. I think I remember this being why the `with pq.ParquetWriter` was the outermost element of this looping construct.

Reply:
Reverted to keeping the `ParquetWriter` at the top level, and only writing to the `parquet/` directory.